packages/android-app/android/build.gradle

```diff
@@ -23,8 +22,7 @@ allprojects {
         jcenter()
 
         maven {
-            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-            url "$rootDir/../../../node_modules/react-native/android"
+            url "$rootDir/../assets/react-native-modules"
```

Comment:
shall we call this different? they're not really assets, more like repository or similar

Reply:
51bbdd0
This will be published to maven soon